flexmeasures/data/models/forecasting/__init__.py

```diff
@@ -25,14 +20,7 @@
 def lookup_model_specs_configurator(
     model_search_term: str = "linear-OLS",
 ) -> Callable[
-    [
-        Union[Asset, Market, WeatherSensor],
-        datetime,
-        datetime,
-        timedelta,
-        Optional[timedelta],
-        Optional[dict],
-    ],
+    ...,  # Annotation requires Python 3.10 (specifically, ParamSpec from PEP 612), see model_spec_factory.create_initial_model_specs for function signature
```

Comment:
I don't understand what to do with this comment.

Reply:
I rephrased my comment, hopefully that clears it up. The former annotation has always troubled my IDE, and now I investigated why. The former annotation had also not been kept up to date.